cmake/CMakeSettings.json

```diff
@@ -0,0 +1,62 @@
+{
+  "configurations": [
+    {
+      "name": "WinML-Debug",
+      "generator": "Ninja",
+      "configurationType": "Debug",
+      "inheritEnvironments": [ "msvc_x64_x64" ],
+      "buildRoot": "${projectDir}\\..\\build\\${name}",
+      "installRoot": "${projectDir}\\..\\build\\${name}\\install",
+      "cmakeCommandArgs": "",
+      "buildCommandArgs": "",
+      "ctestCommandArgs": "",
+      "variables": [
+        {
+          "name": "onnxruntime_USE_WINML",
+          "value": "True",
+          "type": "BOOL"
+        },
+        {
+          "name": "onnxruntime_USE_DML",
+          "value": "True",
+          "type": "BOOL"
+        },
+        {
+          "name": "onnxruntime_BUILD_SHARED_LIB",
+          "value": "True",
+          "type": "BOOL"
+        }
+      ],
+      "cmakeToolchain": "${projectDir}\\wcos_toolchain.cmake"
+    },
+    {
+      "name": "WinML-RelWithDebInfo",
+      "generator": "Ninja",
+      "configurationType": "RelWithDebInfo",
+      "inheritEnvironments": [ "msvc_x64_x64" ],
+      "buildRoot": "${projectDir}\\..\\build\\${name}",
+      "installRoot": "${projectDir}\\..\\build\\${name}\\install",
+      "cmakeCommandArgs": "",
+      "buildCommandArgs": "",
+      "ctestCommandArgs": "",
+      "variables": [
+        {
+          "name": "onnxruntime_USE_WINML",
+          "value": "True",
+          "type": "BOOL"
+        },
+        {
+          "name": "onnxruntime_USE_DML",
+          "value": "True",
+          "type": "BOOL"
+        },
+        {
+          "name": "onnxruntime_BUILD_SHARED_LIB",
+          "value": "True",
+          "type": "BOOL"
+        }
+      ],
+      "cmakeToolchain": "${projectDir}\\wcos_toolchain.cmake"
+    }
+  ]
+}
```

Comment:
is the settings file intentional? looks like it might be user-specific

Reply:
Intentional. It sets default configurations to show when you open the CMake project in VS (like normal VS projects have Debug/Release and platform choices, when using the VS CMake integration you can set default configs in the CMakeSettings.json file). Here I added a couple of configs for WCOS.

I'm going to show to the team the workflow using native VS integration vs CMake command line vs build.py and we can discuss what works best.